.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,53 +1,56 @@
 name: Bug Report
-description: File a bug report
-title: "[Bug]: "
+description: Report a bug
 labels: ["bug"]
 
 body:
   - type: textarea
-    id: description
     attributes:
-      label: Describe the bug
-      description: A clear and concise description of what the bug is.
-      placeholder: What went wrong?
+      label: What happened?
+      description: Please provide as much information as possible, this helps us address the issue.
+    validations:
+      required: true
   - type: textarea
-    id: reproduce
     attributes:
-      label: Steps to reproduce
-      description: |
-        Steps to reproduce the behavior:
-
-        1. Step 1
-        2. Step 2
-        3. ...
-        4. See error
-      placeholder: How can we replicate the issue?
+      label: What did you expect to happen?
+    validations:
+      required: true
   - type: textarea
-    id: modelused
     attributes:
-      label: Model Used
-      description: A description of the model that was used when the error was encountered
+      label: How can we reproduce it (as minimally and precisely as possible)?
+      description: Please provide steps to reproduce. Provide code that can be run if possible.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: AutoGen version
+      description: What version or commit of the library was used
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Which package was this bug in
+      options:
+        - Core
+        - AgentChat
+        - Extensions
+        - AutoGen Studio
+        - Magentic One
+        - AutoGen Bench
+        - Other
+        - I don't know
```

Comment:
Have we exhausted the choices with the above including "Other"?

Reply:
Think we should remove other "I don't know" for people that aren't sure?